crawler/containers.py

```diff
@@ -54,7 +54,7 @@ def poll_containers(timeout, user_list='ALL', host_namespace='',
     :return: a list of Container objects
     """
     # XXX: we only support polling docker containers
-    return poll_docker_containers(timeout, host_namespace, user_list)
+    return poll_docker_containers(timeout, user_list, host_namespace)
```

Comment:
oh man, really?!!

In that case, please change this to poll_docker_containers(timeout, user_list=user_list, host_namespace=host_namespace)